_content/en/01-context/04-yes-this-is-bad.md

```diff
@@ -1,3 +1,3 @@
 # Yes; this is bad
 
-Don’t panic but do not brush off the public health warnings of Coronavirus. Emerging consensus is that containment might have been possible a few weeks ago, but is [no longer realistic](https://twitter.com/uwmnewsroom/status/1236020906956189696). Although China [bought us all time, we wasted it](https://twitter.com/florian_krammer/status/1236344865924972545). We are entering the [pandemic phase which will be followed by seasonal recurrence of the disease](https://twitter.com/NAChristakis/status/1235983934187544578) unless and until we have a vaccine (which may still be [18-24 months away, if we are lucky](https://www.politico.com/news/2020/03/05/coronavirus-trump-vaccine-rhetoric-121796?nname=playbook&nid=0000014f-1646-d88f-a1cf-5f46b7bd0000&nrid=0000014e-f0fe-dd93-ad7f-f8ff7e290000&nlid=630318)).
+Don’t panic but do not brush off the public health warnings of Coronavirus. Emerging consensus is that containment might have been possible a few weeks ago, but is [no longer realistic](https://twitter.com/uwmnewsroom/status/1236020906956189696). Although China [bought us all time, we did not use it effectively](https://twitter.com/florian_krammer/status/1236344865924972545). We are entering the [pandemic phase which will be followed by seasonal recurrence of the disease](https://twitter.com/NAChristakis/status/1235983934187544578) unless and until we have a vaccine (which may still be [18-24 months away, if we are lucky](https://www.politico.com/news/2020/03/05/coronavirus-trump-vaccine-rhetoric-121796?nname=playbook&nid=0000014f-1646-d88f-a1cf-5f46b7bd0000&nrid=0000014e-f0fe-dd93-ad7f-f8ff7e290000&nlid=630318)). Further, recent evidence suggests that [asymptomatic people carrying the virus are spreading it](https://www.cnn.com/2020/03/14/health/coronavirus-asymptomatic-spread/index.html). This makes it much more difficult to contain and social distancing measures even more critical.
```

Comment:
Can we revise 'We are entering the [pandemic phase....' to 'We have entered the [pandemic phase....'

Reply:
@nicolevasilevsky I just went ahead and applied your suggestion